transport/stunnel/client.go

```diff
@@ -0,0 +1,261 @@
+package stunnel
+
+import (
+	"bytes"
+	"context"
+	"strconv"
+	"text/template"
+
+	"github.com/backube/pvc-transfer/transport"
+	"github.com/backube/pvc-transfer/transport/tls/certs"
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const clientListenPort = 6443
+
+const (
+	stunnelClientConfTemplate = `
+ pid =
+ sslVersion = TLSv1.3
+ client = yes
+ syslog = no
+ output = /dev/stdout
+ [rsync]
```

Comment:
To match what we did in server?
```suggestion
 [transfer]
```

Reply:
good catch, thanks!